doc/protocols.rst

```diff
@@ -216,3 +216,12 @@ References: `Wikipedia <https://en.wikipedia.org/wiki/Sina_Weibo>`_.
 IEEE 1588-2008 Precision Time Protocol (PTP) Version 2.
 
 References: `Protocol Specs: <https://standards.ieee.org/ieee/1588/4355/>`_.
+
+
+.. _Proto 72:
+
+`NDPI_PROTOCOL_HART_IP`
+============================
+Highway Addressable Remote Transducer over IP
+
+References: `Protocol Specs: <https://library.fieldcommgroup.org/20085/TS20085>`_.
```

Comment:
Could you move this new section up, keeping the protocol numbers ordered? 72/HART should be the first of the list....

Reply:
Done